packages/framework-core/src/services/event-store.ts

```diff
@@ -73,6 +73,18 @@ export class EventStore {
     }
   }
 
+  @Trace(TraceActionTypes.CUSTOM)
+  public async storeDispatchedEvent(eventEnvelope: EventEnvelope) {
+    const logger = getLogger(this.config, 'EventStore#storeDispatchedEvent')
+    try {
+      logger.debug('Storing event in the dispatched event store:', eventEnvelope)
+      return await this.config.provider.events.storeDispatched(eventEnvelope, this.config)
+    } catch (e) {
+      logger.error('Could not store dispatched event: ', eventEnvelope)
+      return
```

Comment:
I think it is a good idea what you are doing here. I mean, not to stop the processing of the event if there was an error. I would just return true and say so in the log. Something like: 
```suggestion
      logger.error('Could not store dispatched event. Continue its processing.', eventEnvelope)
      return true
```